packages/world/src/modules/init/implementations/CallWithSignatureSystem/ECDSA.sol

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v5.0.0) (utils/cryptography/ECDSA.sol)
```

Comment:
did we decide it was better to copy/paste these here instead of importing from OZ as a dep?

Reply:
I think the original reason was that the OZ package was a pretty heavy dependency and we only needed very few files from it. At this point we do have a couple vendored OZ files in the world package (`ERC165Checker`, `IERC1967`, `Proxy`, `StorageSlot`) but I don't think it's worth changing our approach now. MUD is audited as a whole and the auditors can easily check that the files match the original sources. Changing paths of a Solidity package is always annoying because it would break things downstream if the files are imported since there are no explicit exports/remappings.

Reply:
I _hope_ no one has imported these directly 🙈 wonder if we should start putting not-for-external-use source files into `internal` dirs, like we've started doing for TS exports